src/cmds/restic/integration_test.go

```diff
@@ -516,23 +516,23 @@ func TestBackupExclude(t *testing.T) {
 		testRunBackup(t, []string{datadir}, opts, gopts)
 		snapshots, snapshotID := lastSnapshot(snapshots, loadSnapshotMap(t, gopts))
 		files := testRunLs(t, gopts, snapshotID)
-		Assert(t, includes(files, filepath.Join("testdata", "foo.tar.gz")),
+		Assert(t, includes(files, filepath.Join("/testdata", "foo.tar.gz")),
```

Comment:
Huh, that's odd. It seems that `filepath.Join()` on Windows automatically converts `/testdata` to `\testdata`. That's why the test does not fail on Windows. I'd still prefer having the filepath separator in, for clarity. Thanks!